pkg/minikube/constants/constants.go

```diff
@@ -44,7 +43,7 @@ const (
 	DefaultDiskSize = "20g"
 )
 
-var DefaultIsoUrl = "https://github.com/minishift/minishift/releases/download/v" + version.GetVersion() + "/boot2docker.iso"
+var DefaultIsoUrl = "https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.0/minishift-b2d.iso"
```

Comment:
+1 to the idea of automatically fetching the latest release. I am only concerned with the call to api.github.com. Users are already facing some issues with it with respect to number of calls being made.